### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jeremyrader**

### CODE REVIEW

**Comment by thealexbaron** [comment]
I am commenting on the whole pr.

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# github-slack-testing
\ No newline at end of file
+# github-slack-testing
+
+Is it Friday yet?
```
Comment (thealexbaron) [inline README.md]: Yes, it is Friday. This is a line level comment.

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# github-slack-testing
\ No newline at end of file
+# github-slack-testing
```
Comment (thealexbaron) [inline README.md]: Cool code comment

Comment (jeremyr8r) [inline README.md]: no, its not cool

Comment (jeremyr8r) [inline README.md]: its really really not cool

Comment (jeremyr8r) [inline README.md]: in fact, it sucks

Comment (jeremyr8r) [inline README.md]: big time

**Pull request closed without merging**